modules/ztp-setting-managed-bare-metal-host-kernel-parameters.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.adoc
+
+[id="setting-managed-bare-metal-host-kernel-parameters_{context}"]
+= Setting managed bare-metal host kernel parameters
+
+Specify kernel parameters for managed bare-metal hosts to customize the host's boot process. This is useful for clusters with specific environmental requirements. For example, specify the `rd.net.timeout.carrier` kernel parameter to facilitate static networking or to receive a DHCP address before downloading the root file system. 
+
+[NOTE]
+====
+In {product-title} {product-version}, you can only add kernel parameters. You can not replace or delete kernel parameters.
+====
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (oc).
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+* You have generated the installation and configuration custom resources(CRs).
+
+.Procedure
+
+* Edit the `spec.kernelArguments` specifications in the `InfraEnv` CR to add kernel values:
```

Comment:
In Step 5 of the manual install, you generate the day-0 installation CRs by processing the SiteConfig CR. It is then possible to edit the InfraEnv.yaml file. 
What is the workdflow for  editing an InfraEnv.yml file when using the [GitOps approach](https://55012--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.html#ztp-deploying-a-site_ztp-deploying-far-edge-sites)? Do you need to create an extra-manifests folder to hold a custom InfraEnv.yaml file?  This folder is mentioned in step 3.d [here](https://55012--docspreview.netlify.app/openshift-enterprise/latest/scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-sites.html#ztp-deploying-a-site_ztp-deploying-far-edge-sites). Or is there some other way?